README.md

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
Nit: unintentional newline?

Reply:
Weird. Not sure how that happened, but fixed now.